Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,70 @@
+directly on the host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external
+loadbalancer, ...).
+
+The host network mode can be enabled via Helm flag
+``gatewayAPI.hostNetwork.enabled=true``.
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and higher than ``1023``.
+
+.. warning::
+    * Enabling the Cilium GatewayAPI host network mode automatically disables the LoadBalancer/NodePort type service mode. They are mutually exclusive.
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+    * Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available.
+    * It's mandatory to use ports higher than ``1023`` as the Cilium L7 proxy process `does not have any Linux capabilities by default`_ (so no ``CAP_NET_BIND_SERVICE``) and is therefore not allowed to listen on privileged ports. If you choose a port equal to or lower than ``1023``, ensure that ``envoy.securityContext.capabilities.keepCapNetBindService=true`` is configured and to add the capability ``NET_BIND_SERVICE`` to the respective :ref:`Cilium Envoy container via Helm values<envoy>`:
+
+        * Standalone DaemonSet mode: ``envoy.securityContext.capabilities.envoy``
+        * Embedded mode: ``securityContext.capabilities.ciliumAgent``
+
+.. _does not have any Linux capabilities by default: https://github.com/cilium/proxy/pull/315
+
+Furthermore, it's supported to expose the Cilium GatewayAPI Envoy
+listener only on specific nodes. This only works in combination with the
+host network mode and can be configured via a node label selector in the Helm
+values. Here is a complete host network mode example configuration for
+Cilium GatewayAPI:
+
+.. code-block:: yaml
+
+    gatewayAPI:
+      enabled: true
+      hostNetwork:
+        enabled: true
+        nodes:
+          matchLabels:
+            role: infra
+            component: gateway-api
+    # -- Optional: Allow privileged port bindings (not recommended)
+    #
+    # Configure the following to allow privileged port bindings in host network mode.
+    #
+    # For Cilium Envoy standalone DaemonSet mode:
+    envoy:
+      enabled: true
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+          envoy:
+          # Add NET_BIND_SERVICE to the list (keep the others!)
+          - NET_BIND_SERVICE
+    # For Cilium Envoy embedded mode:
+    envoy:
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+    securityContext:
+      capabilities:
+        ciliumAgent:
+        # Add NET_BIND_SERVICE to the list (keep the others!)
+        - NET_BIND_SERVICE
+
+An empty selector selects all nodes and continues to expose the functionality
```

Comment:
Maybe prepend the sentence:

> This will deploy the Gateway API Envoy listener only on the Cilium Nodes matching the configured labels.

Followed by `An empty selector ...`

Reply:
Done.